InvenTree/templates/js/translated/forms.js

```diff
@@ -493,6 +493,30 @@ function constructFormBody(fields, options) {
         html += options.header_html;
     }
 
+    // process every field by recursively walking down nested fields
+    const processField = (name, field, optionsField) => {
+        if (field.type === "nested object") {
+            for (const [k, v] of Object.entries(field.children)) {
+                processField(`${name}__${k}`, v, optionsField.children[k]);
```

Comment:
@wolflu05 this has broken existing form handling. e.g. try to edit an existing purchase order:

![image](https://github.com/inventree/InvenTree/assets/10080325/aa2d694c-94ab-44f3-b2d0-5cdaa2222227)

Please look into this one ASAP